packages/react-router/test-app/cypress/support/commands.js

```diff
@@ -102,7 +102,7 @@ Cypress.Commands.add('ionMenuNav', (contains) => {
 
 Cypress.Commands.add('ionTabClick', (tabText) => {
   // TODO: figure out how to get rid of this wait. Switching tabs after a forward nav to a details page needs it
-  cy.wait(250);
+  cy.wait(500);
```

Comment:
Should we add a tech debt ticket here?

Reply:
Alternatively, I could make it an explicit requirement in the new routing design doc that arbitrary timeouts should not be needed when testing the router.

Reply:
Adding a note to the design doc works for me 👍